docs/bokeh/source/docs/user_guide/basic/areas.rst

```diff
@@ -34,8 +34,8 @@ glyph function:
 .. bokeh-plot:: __REPO__/examples/basic/areas/block.py
     :source-position: above
 
-Rotatable
-~~~~~~~~~
+Rectangles
+~~~~~~~~~~
```

Comment:
This should be reverted, for two reasons

* The entire overall section is already "Rectangles", to cover all the rect-like glyphs, so this would make the section hierarchy become: "Rectangles -> Rectangles" which seems awkward
* The main characteristic that distinguishes `rect` glyphs from all other rect-like area glyphs, is that they are center-rotatable, rather than axis-aligned.